### ISSUE TITLE
Updating the HLT lessons

### ISSUE THREAD

**Issue by alexpearce** [lhcb/second-analysis-steps#20]
This is mostly for @roelaaij 😄

Is the [introduction to the HLT](https://lhcb.github.io/second-analysis-steps/18-hlt-intro.html) up-to-date? Is all the content there that we want to teach?

One thing is to change the text about Run 2, for example

> In Run II the reconstructions used in HLT2 and offline will be identical

Also things like if the HLT1 pT threshold may have changed in Run 2 (currently stated that HLT1 only reconstructs pT > 500 MeV).

There are some broken links as well, like in the sentence:

> We will use the script we created earlier as a starting point, and the file you downloaded before.

Would you have time to go through the text and update it @roelaaij?

**Comment by roelaaij** [comment]
Yes, I'm planning to go through it. Can I view your Turbo material somewhere so I can try to ensure things match?

**Comment by alexpearce** [comment]
There is nothing yet 😞 But now we can discuss how we will do it.

I can just go through ‘how to make ntuples from Turbo stream µDSTs’. Or I could talk for a little bit about why Turbo exists and how it's useful. Or you could include that information in the HLT part.

I don't mind too much how we do it; we'll be doing the Turbo-ntuple-making stuff after the HLT session, so it shouldn't matter for the students at least.
